code/game/turfs/open/floor/mineral_floor.dm

```diff
@@ -58,6 +58,17 @@
 	if(exposed_temperature > 300)
 		PlasmaBurn(exposed_temperature)
 
+//Plasma floor that can't be removed, for disco inferno
+
+/turf/open/floor/mineral/plasma/disco
+	name = "plasma floor"
+	icon_state = "plasma"
+	floor_tile = /obj/item/stack/tile/mineral/plasma
+	icons = list("plasma","plasma_dam")
```

Comment:
youre just repeating stuff from the parent, might as well just remove this whole segment

Reply:
Very good point